lib/utils/embargoedAssets.js

```diff
@@ -31,7 +31,11 @@ async function createCachedAssetKey (host, accessToken, spaceId, environmentId,
     try {
       const assetKeyPromise = createAssetKey(host, accessToken, spaceId, environmentId, expiresAtMs)
 
-      cacheItem = { expiresAtMs, promise: assetKeyPromise }
+      const result = await assetKeyPromise
+      const promiseWrapper = new Promise((resolve) => {
```

Comment:
isn't `result.json()` already a promise?

Reply:
I don't think so. https://www.npmjs.com/package/node-fetch#bodyjson
It's just a parsing method